server/odc-service/src/main/java/com/oceanbase/odc/service/task/caller/NativeK8sJobClient.java

```diff
@@ -51,15 +60,27 @@
  */
 public class NativeK8sJobClient implements K8sJobClient {
 
-    public NativeK8sJobClient(String k8sClusterUrl) throws IOException {
-        ApiClient apiClient = Config.defaultClient();
+    public NativeK8sJobClient(TaskFrameworkProperties.K8sProperties k8sProperties) throws IOException {
+        ApiClient apiClient = null;
+        if (StringUtils.isNotBlank(k8sProperties.getKubeConfig())) {
+            byte[] kubeConfigBytes = EncodeUtils.base64DecodeFromString(k8sProperties.getKubeConfig());
+            Verify.notNull(kubeConfigBytes, "kube config");
+            try (Reader targetReader = new InputStreamReader(new ByteArrayInputStream(kubeConfigBytes))) {
+                KubeConfig kubeConfig = KubeConfig.loadKubeConfig(targetReader);
+                apiClient = ClientBuilder.kubeconfig(kubeConfig).build();
+            }
+        } else {
+            apiClient = Config.defaultClient().setBasePath(k8sProperties.getUrl());
+        }
+        Verify.notNull(apiClient, "k8s api client");
         apiClient.setHttpClient(apiClient
                 .getHttpClient()
                 .newBuilder()
-                .readTimeout(Duration.ZERO)
+                .readTimeout(60000, TimeUnit.MILLISECONDS)
```

Comment:
Avoid using magic number, better to declare a static field.

Reply:
done